geometry_msgs/Point[] corners
uint8 id
